packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5521,15 +5530,15 @@ Certificate chain description for verifying assertions from the Smart Card.
 
 #### IdP type policy actions
 
-| Type         | User Provisioning Actions     | Group Provisioning Actions            | Account Link Actions          | Account Link Filters  |
+| IdP Type         | User Provisioning Actions     | Group Provisioning Actions            | Account Link Actions          | Account Link Filters  |
 | ------------ | ----------------------------- | ------------------------------------- | ----------------------------- | --------------------  |
-| `FACEBOOK`   | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    | `AUTO`, `CALLOUT`, `DISABLED` | `groups`              |
-| `GOOGLE`     | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    | `AUTO`, `CALLOUT`, `DISABLED` | `groups`              |
-| `LINKEDIN`   | `AUTO`, `CALLOUT`, `DISABLED` | `NONE` or `ASSIGN`                    | `AUTO`, `CALLOUT`, `DISABLED` | `groups`              |
+| [[all social IdP types *]](#social-idp-type-policy-actions)   | `AUTO`, `DISABLED` | `NONE` or `ASSIGN`                    | `AUTO`, `DISABLED` | `groups`              |
 | `SAML2`      | `AUTO` or `DISABLED`          | `NONE`, `ASSIGN`, `APPEND`, or `SYNC` | `AUTO`, `DISABLED`            | `groups`              |
 | `X509`       | `DISABLED`                    | No support for JIT provisioning       |                               |                       |
 
-> **Note:** `CALLOUT` is a <ApiLifecycle access="deprecated" /> User provisioning action and Account Link action.
+##### Social IdP type policy actions *
+
+All social IdPs (such as `FACEBOOK`, `LINKEDIN`, `GOOGLE`, and so on) support the same User Provisioning Actions, Group Provisioning Actions, Account Link Actions, and Account Link Filters.
```

Comment:
I think we should have a complete list of all social IdPs. Or perhaps it would be easier to say that social IdP = any IdP that is not `SAML2`, `X509`, `OIDC` (I'm not sure if this is correct).

Reply:
Yes, I like the exception option better than listing all of the social IdPs. The goal is to just update the IdP type table for new IdPs. This table shouldn't change unless there's some policy exception.